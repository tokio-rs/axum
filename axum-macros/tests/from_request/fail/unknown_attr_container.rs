use axum_macros::FromRequest;

#[derive(FromRequest)]
#[from_request(foo)]
struct Extractor;

fn main() {}
