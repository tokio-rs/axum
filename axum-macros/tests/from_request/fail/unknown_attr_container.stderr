error: expected one of: `via`, `rejection`, `state`
 --> tests/from_request/fail/unknown_attr_container.rs:4:16
  |
4 | #[from_request(foo)]
  |                ^^^
